content/userguide/OpenPGP.md

```diff
@@ -19,112 +19,59 @@ The private key can be stored in CanoKey, or you can use CanoKey to generate a O
+* Admin PIN: Default is 12345678, minimum length is 8, maximum length is 64
+* Reset Code: Default is empty, minimum length is 8, maximum length is 64
+* Signature PIN: forced (PIN verification required for each signature)
+* Touch Policy: SIG, DEC, AUT are off
 * Touch Cache Time: 0
 
-## Touch Policy
+### Touch Policy
 
-There are three key slots for OpenPGP, namely Signature key (SIG), Encryption key (DEC) and Authentication key (AUT). You may turn ON or OFF touch policies for SIG, DEC, AUT in the admin applet in the web console or via the `gpg` command. The value of touch cache time is in between 0 and 255 seconds (0 means no cache).
+{{% notice note %}}
+Touch policy is only effective when using the USB interface.
+{{% /notice %}}
 
-### Firmware Version <= 1.4
+OpenPGP supports up to 3 keys: signature key (SIG), encryption key (DEC), and authentication key (AUT). Depending on the firmware version, you can set the touch policy for SIG, DEC, and AUT in the CanoKey Console or via the `gpg` command. The value of touch cache time ranges from 0 to 255 seconds (0 means no cache).
 
-Touch policy is configured through the admin applet. The technical details can be found in [https://docs.canokeys.org/development/protocols/admin/](https://docs.canokeys.org/development/protocols/admin/).
+#### Firmware Version <= 1.4
 
-### Firmware Version >= 1.5
+Please use the "Settings" application in the CanoKey Console to modify the touch policy.
 
-Touch policy is implemented using the User Interaction Flag (Part 4.4.3.6 of the OpenPGP specification). Use a recent GnuPG to configure it.
+#### Firmware Version >= 1.5
 
-**Touch policy is only applicable when using the USB interface.**
+Please use GnuPG to modify the touch policy.
 
-## PIN Policy
+### PIN Policy
 
-For DEC and AUT, after successfully verifying the PIN, it is always valid for the whole power-up.
+For DEC and AUT keys, after the PIN verification is successful, verification will not be required again until CanoKey is disconnected and reinserted.
 
-For SIG, if the flag `forcesig` is on, the PIN is requested for each signature; otherwise, PIN is only requested for the first signature after power-up.
+For SIG, if `forcesig` is on, a PIN is required for each signature; otherwise, a PIN is only required for the first signature after power-on.
 
-## Using GnuPG
+## Common Operations
 
-Please refer to the [GNU Privacy Handbook](https://gnupg.org/gph/en/manual.html) at the moment.
+## FAQs
 
-You may also refer to [https://github.com/drduh/YubiKey-Guide](https://github.com/drduh/YubiKey-Guide).
+### GnuPG and PC/SC Conflict
 
-### Cheatsheet
+GnuPG, by default, uses its own implementation ([scdaemon](https://www.gnupg.org/documentation/manuals/gnupg/Invoking-SCDAEMON.html)) to access smart cards including CanoKey, which conflicts with PC/SC. For details, see: <https://ludovicrousseau.blogspot.com/2019/06/gnupg-and-pcsc-conflicts.html>.
 
-```
-# card related
```

Comment:
Section removed?

Reply:
Yes, I believe that we need a better tutorial for common use case. A simple cheatsheet is not helpful.